### DISCUSSION TITLE
Product Variants release ?

### DISCUSSION THREAD

**Discussion by baodv** [vanilophp/framework#157] [General]
Product Variants release version 3.0 , but i didn't find

**Comment by fulopattila122** [comment]
How did you download Vanilo? Where did you look for the variants?

**Comment by baodv** [comment]
I installation document https://vanilo.io/docs/3.x/installation
I find : https://vanilo.io/docs/3.x/product-variants
But when i install vanilo then  variants not found

In github https://github.com/vanilophp , i can'n look

**Reply by fulopattila122** [reply]
- What version do you have installed exactly? (composer show vanilo/framework)
- How did you try to locate the variants?

Please provide with more details in order to be able to help you

**Reply by fulopattila122** [reply]
What does "I can'n look" mean?

**Comment by baodv** [comment]
Where can I find variants?
